repository/cowsay/operator/operator.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: kudo.dev/v1beta1
+name: "cowsay"
+version: "0.1.0"
+kubernetesVersion: 1.13.0
+maintainers:
+  - name: zen-dog
+    email: <adukhovniy@d2iq.com>
```

```suggestion
    email: <adukhovniy@D2iQ.com>
```
I think that's the spelling marketing wants us to use.